source/extensions/omni.isaac.lab/omni/isaac/lab/sim/spawners/sensors/sensors_cfg.py

```diff
@@ -71,6 +73,72 @@ class PinholeCameraCfg(SpawnerCfg):
     the camera output on the GUI and not accidentally moving the camera through the GUI interactions.
     """
 
+    @classmethod
+    def from_intrinsic_matrix(
+        cls,
+        intrinsic_matrix: list[float],
+        width: int,
+        height: int,
+        projection_type: str = "pinhole",
+        clipping_range: tuple[float, float] = (0.01, 1e6),
+        focal_length: float = 24.0,
+        focus_distance: float = 400.0,
+        f_stop: float = 0.0,
+        horizontal_aperture: float = 20.955,
+        horizontal_aperture_offset: float = 0.0,
+        vertical_aperture_offset: float = 0.0,
```

Comment:
These are exposed but not used right? 🤔

Reply:
did the not implemented error but the arguments are technically set inside the simulator, so I would assume they are used.

Reply:
okay they are not used, I will throw a general warning that currently these parameters don't do anything!

Reply:
see latest changes